### ISSUE TITLE
Error with estimatePsite()

### ISSUE THREAD

**Issue by JC-therea** [jianhong/ribosomeProfilingQC#5]
Hi, I am having an error with this library. I am currently using, one specific annotation which is not on NCBI. I was able to forge its genome but when I want to calculate the psite error outputs the following error:

```
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'x' in selecting a method for function 'intersect': The style does not have a compatible entry for the species supported by
  Seqname. Please see genomeStyles() for supported species/style
```
Probably is related with the function `seqlevelsStyle()`. Is there any solution? Could you let me know if you plan to modify that specific part?

Thank you very much for your help.

**Comment by jianhong** [comment]
I update the development version with ignore.seqlevelsStyle parameter. Try to update and set the ignore.seqlevelsStyle to TRUE. Let me know if it does not work for you.

**Comment by JC-therea** [comment]
Hello, thanks for the update! However, there are still some scripts that do not have the option included. For example, I used
`estimatePsite()` and now has the option `ignore.seqlevelsStyle` but still doesn't work for me.  I noticed that inside the `R/estimatePsite.R` function the function `assignReadingFrame()` is using the default option of `ignore.seqlevelsStyle` (which is FALSE) and I think that, that part is producing the error (which is the same as before). For the moment, I am only using the functions `estimatePsite()` which has that problem that I mentioned, and `assignReadingFrame()` which works well, I don't know about the rest functions.

Again thank you very much for improving your software with my suggestions.
